src/SpillRegisters_X64.S

```diff
@@ -0,0 +1,27 @@
+.text
+
+.extern scan_stack
+.globl spill_registers
+
+spill_registers:
+    // Push all callee-saves registers to get them on the stack for conservative
```

Comment:
Typo: "callee-save"

Reply:
Did you want to document why it's only the callee save registers that we care about?

Reply:
Sure, done.

Reply:
LGTM